core/thread.py

```diff
@@ -336,6 +343,72 @@ def _close_after(self, closer, silent, delete_channel, message):
                 if str(message_id) == str(embed.author.url).split("/")[-1]:
                     return msg
 
+    async def _grab_timeout(
+            self
+    ) -> typing.Union[None, isodate.duration.Duration, timedelta]:
```

Comment:
do this instead in one line:
```python
async def _grab_timeout(self) -> typing.Union[None, isodate.duration.Duration, timedelta]:
```

Also, rename the function to `_fetch_timeout` 😛

Reply:
I was following PEP8. Are we using a different code style? 😅

Reply:
We follow our code style lul

whatever looks/works best i guess..

Reply:
Just use black for code formatting.

Reply:
@fourjr I committed that into one line, but the second black formatting commit reverted it back.